bulletroboy/exoforce/exoforce.py

```diff
@@ -10,12 +9,17 @@
 from rclpy.node import Node
 from roboy_control_msgs.msg import CageState, EndEffector, ViaPoint as ViaPointMsg, MuscleUnit as MuscleUnitMsg
 from roboy_control_msgs.srv import GetCageEndEffectors
+from roboy_simulation_msgs.msg import Collision
+from geometry_msgs.msg import PoseStamped
 from geometry_msgs.msg import Point
 
+from ..utils.force_decomposition import decompose_force_ef_to_tendons
```

Comment:
do relative imports always work? independent from current directory

Reply:
yes, they are relative to the script directory